app/render/index.ts

```diff
@@ -46,14 +50,14 @@ declare global {
 	interface File {
 		path: string;
 	}
-}
 
-const nightPDF = (async function () {
+}
+await (async function () {
 	console.log("loading");
 	let _pdfElement: HTMLIFrameElement;
```

Comment:
this no longer exists which breaks the `file-print` function.

Reply:
No probs, I can revert that change :)

Reply:
Looking back, I'm not even sure how this change occurred, it definitely wasn't intentional haha, I think I accidentally hit a button in VSCode

Reply:
> _pdfElement used to be needed for printing since it used that to print the iframe, looking at the webview electron docs there is a webview.print() function

Yeah I've fixed and pushed the update, just for reference

`webview.print()` did not work for me, it ended up making a PDF like this:  (where the chrome / scrollbars are actually inside the 1 page PDF)

<img width="630" alt="image" src="https://github.com/Lunarequest/NightPDF/assets/75656/1c823fe2-e3f7-4b51-b202-203e1ff7b3df">

Instead, I leveraged the `webview.executeJavaScript()` method to invoke the `window.print()` method (which is overwritten by `pdfjs` within the webview.